en/docs/fragments/pickup-manager-saml.md

```diff
@@ -2,7 +2,70 @@
 
 ### Register a service provider
 
-insert portal content
+!!! note "Important"
+
+    SAML2 POST Binding requires CORS configs set up. Before configuring the service provider, make sure you add the following configurations to the `<IS_HOME>/repository/conf/deployment.toml` file to allow HTTP POST requests. 
+
+    ```toml
+    [cors]
+    allow_generic_http_requests = true
+    allow_any_origin = false
+    allowed_origins = [
+        "http://localhost:8080"
+    ]
+    allow_subdomains = false
+    supported_methods = [
+        "GET",
+        "POST",
+        "HEAD",
+        "OPTIONS"
+    ]
+    support_any_header = true
+    supported_headers = []
+    exposed_headers = []
+    supports_credentials = true
+    max_age = 3600
+    tag_requests = false
+    ```
```

Comment:
use {!fragments/cors-config.md!} instead. Also, let's have this under a separate topic. ex: https://github.com/wso2/docs-is/pull/2506/files#diff-5369a3b41329f57f33c091f53d582ff6181c9664ff60941549e6b98a0bfa5e83R43-R45